sled-agent/src/rack_setup/service.rs

```diff
@@ -867,6 +859,23 @@ impl ServiceInner {
                 .await?;
         }
 
+        // Save the relevant network config in the bootstore. We want this to
+        // happen before we `initialize_sleds` so each scrimlet (including us)
+        // can use its normal boot path of "read network config for our switch
+        // from the bootstore".
+        let early_network_config = EarlyNetworkConfig {
+            generation: 1,
+            rack_subnet: config.rack_subnet,
+            ntp_servers: config.ntp_servers.clone(),
+            rack_network_config: config.rack_network_config.clone(),
+        };
+        info!(self.log, "Writing Rack Network Configuration to bootstore");
+        bootstore.update_network_config(early_network_config.into()).await?;
+        // TODO: Wait for bootstore to sync to all `plan.sleds`? Every
```

Comment:
I'd remove this comment since we poll in sled-agent start *and* we have an option for deployments without `rack_network_config`

Reply:
removed in 05fb1ded